guide/contribute/case-studies.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: guide
+title: Case Studies
+description: How to contribute case studies to the guide
+nav_order: 10
+parent: Contribute to guide
+permalink: /guide/contribute/case-studies/
+main_classes: -no-top-padding
+image: https://bitcoin.design/assets/images/guide/contribute/contribute-preview.jpg
+---
+
+{% include picture.html
+   image = "assets/images/guide/contribute/case-studies/CasestudiesHeader.png"
+   retina = "assets/images/guide/contribute/case-studies/CaseStudiesHeader2X.png"
+   mobile = "assets/images/guide/contribute/case-studies/CaseStudiesHeaderMobile.png"
+   mobileRetina = "assets/images/guide/contribute/case-studies/CasestudiesHeaderMobile2X.png"
+   alt-text = "Image of people reading books"
```

Comment:
The screen reader already knows this is an image, no need to repeat (see [alt text writing tips](https://webaim.org/techniques/alttext/)).